packages/SwingSet/misc-tools/monitor-slog-block-time.py

```diff
@@ -98,7 +101,15 @@ def load_genesis_keys(fn):
 
 def wait_for_block(height):
     while True:
-        out = subprocess.check_output(["agd", "status"]).decode()
+        # TODO: 'agd status' has started failing with messages like:
+        ## diff: node_modules/.cache/agoric/golang-built.sum.new: No such file or directory
+        ## golang/cosmos node_modules/.cache/agoric/golang-built.sum has changed
+        ## mv: cannot stat 'node_modules/.cache/agoric/golang-built.sum.new': No such file or directory
```

Comment:
Are these comments still relevant?

Reply:
I think so? I'd have to remove the sleep-then-retry loop to find out.